### PR TITLE
test dataplane prep targets

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

File: scripts/clone-operator-repo.sh [hunk 1]
```diff
@@ -65,8 +65,15 @@ function git_clone_checkout {
 
     pushd ${OPERATOR_BASE_DIR}
 
-    echo "Cloning repo: git clone ${git_opts} ${branch} ${repo} ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT}"
-    git clone ${git_opts} ${branch} ${repo} ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT}
+    if [ ! -d ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT} ]; then
```
Comment (slagle) [inline scripts/clone-operator-repo.sh:68]: is this change necessary as the checkout should always be deleted by edpm_deploy_cleanup. I don't think this will be an issue any longer with your other Makefile changes.

Comment (fao89) [reply scripts/clone-operator-repo.sh:68]: you are right!

**Review by slagle — COMMENTED** [review]
LGTM

**Comment by ASBishop** [comment]
I encountered the problem, found this patch and tried it, and it worked perfectly. I recommend you remove the DNM and commit message, as it lgtm!
